automated install/basic-install.sh

```diff
@@ -2304,6 +2304,25 @@ main() {
     fi
   fi
 
+  # resolved and dnsmasq can't share port 53. 
+  # resolved needs to remain in place for installer to download needed files
+  # so this change needs to be made after installation is complete, but before resarting dnsmasq/ftl
+
+  # Check if running ubuntu 18.04 bionic beaver, which ships with resolved active on port 53
+  # (This check may need to be broadened for other systems running resolved?)
+  if ( lsb_release -a | grep 'Ubuntu 18.04' &> /dev/null ); then
+    # Running ubuntu 18.04, so check if resolved is running,
+    if (systemctl is-enabled systemd-resolved | grep -c 'enabled' || true); then
```

Comment:
Is `systemctl` guaranteed to reply in English? Mind that there are tons of devices with German, Spanish, Dutch, ... locales and most Linux commands are actually capable of responding in the set locale. We should rely on return codes instead of return strings